### PR TITLE
refactor(app): update Labware Help Modal

### LINKED ISSUE #8325: Move and update copy for Labware Help modal

### PR DESCRIPTION
**Pull request by jerader**
# Overview

closes #8325 

# Changelog

- moves modal to be in `Labware Position Check and Offset Data` section in Labware setup accordion
- updates modal to match [figma](https://www.figma.com/file/JCQNOTTK9tTmYPhRqeOfSj/Milestone-0%3A-Protocol-Upload-Revamp?node-id=3292%3A109)
- renamed `LabwareSetupModal` to `LabwareOffsetModal` to describe the information in the new modal better

# Review requests

- review to make sure styling, text, and links are correct

# Risk assessment

low, behind ff

### CODE REVIEW

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareOffsetModal.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  SPACING_3,
+  SPACING_1,
+} from '@opentrons/components'
+import { Portal } from '../../../../App/portal'
+import styles from '../../styles.css'
+
+const ROBOT_CAL_HELP_ARTICLE =
+  'https://support.opentrons.com/en/articles/3499692-how-positional-calibration-works-on-the-ot-2'
+
+const OFFSET_DATA_HELP_ARTICLE = '#' //  REPLACE WITH ACTUAL LINK
```
Comment (jerader) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareOffsetModal.tsx]: Emily said the article this is supposed to link to doesn't exist yet

**Comment by jerader** [comment]
Hi @emilywools, please review the following and let me know how everything looks! Thanks 😄 
 
Screen Shot 2021-10-20 at 08 56 20
Screen Shot 2021-10-20 at 08 56 43
>

**Comment by emilywools** [comment]
hey Jethary, this looks great! One thing: the spacing before the "learn more about xx" and after should be different. Before the link, the paragraph spacing should be a little bit less so it looks like it belongs with the paragraph above it.

**Comment by jerader** [comment]
> hey Jethary, this looks great! One thing: the spacing before the "learn more about xx" and after should be different. Before the link, the paragraph spacing should be a little bit less so it looks like it belongs with the paragraph above it.

@emilywools, thanks for noticing that! I did have extra spacing. How is this? 
Screen Shot 2021-10-20 at 10 15 21

**Comment by emilywools** [comment]
LGTM!!!

**Review by emilywools — APPROVED** [review]

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareOffsetModal.tsx (hunk 1 above)
Comment (b-cooper) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareOffsetModal.tsx]: I'd add a TODO: IMMEDIATELY to this comment as a reminder that this needs to be addressed before release.

**Review by b-cooper — APPROVED** [review]
Looks good to me. Let's make sure to flag the missing article link as an immediate TODO

**Pull request merged**